google-ads-examples/src/main/java/com/google/ads/googleads/examples/basicoperations/AddResponsiveSearchAd.java

```diff
@@ -0,0 +1,160 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.basicoperations;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v1.common.AdTextAsset;
+import com.google.ads.googleads.v1.common.ResponsiveSearchAdInfo;
+import com.google.ads.googleads.v1.enums.AdGroupAdStatusEnum.AdGroupAdStatus;
+import com.google.ads.googleads.v1.enums.ServedAssetFieldTypeEnum.ServedAssetFieldType;
+import com.google.ads.googleads.v1.errors.GoogleAdsError;
+import com.google.ads.googleads.v1.errors.GoogleAdsException;
+import com.google.ads.googleads.v1.resources.Ad;
+import com.google.ads.googleads.v1.resources.AdGroupAd;
+import com.google.ads.googleads.v1.services.AdGroupAdOperation;
+import com.google.ads.googleads.v1.services.AdGroupAdServiceClient;
+import com.google.ads.googleads.v1.services.MutateAdGroupAdResult;
+import com.google.ads.googleads.v1.services.MutateAdGroupAdsResponse;
+import com.google.ads.googleads.v1.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class AddResponsiveSearchAd {
```

Comment:
Add javadoc explaining what the example does

Reply:
"display" -> "search". Also, this example doesn't use MediaService. Did you copy it from the ResponsiveDisplayAd example, perhaps?